Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -139,10 +139,22 @@ void HotkeyScheduler::Run()
 
   while (!m_stop_requested.IsSet())
   {
-    Common::SleepCurrentThread(1000 / 60);
+    const u64 freelook_update_freq = 200;
+    const u64 interval = SystemTimers::GetTicksPerSecond() / freelook_update_freq;
+    const u64 now = CoreTiming::GetTicks();
 
-    if (Core::GetState() == Core::State::Uninitialized || Core::GetState() == Core::State::Paused)
+    if (now - m_last_freelook_ticked > interval)
+    {
+      m_last_freelook_ticked = now;
+      g_controller_interface.SetCurrentInputChannel(ciface::InputChannel::FreeLook);
       g_controller_interface.UpdateInput();
+      FreeLook::UpdateInput();
+    }
+
+    Common::SleepCurrentThread(1000 / 60);
+
+    g_controller_interface.SetCurrentInputChannel(ciface::InputChannel::Host);
+    g_controller_interface.UpdateInput();
 
     if (!HotkeyManagerEmu::IsEnabled())
```

Comment:
HotkeyManagerEmu::IsEnabled() should not disable freelook IMO.

Reply:
It doesn't.  It only disables the ability to toggle FreeLook on/off via hotkey

Reply:
right, you move the freelook update above, nevermind.